intro-to-tests/intro-to-pytest.md

```diff
@@ -60,11 +81,12 @@ Take care to notice the following:
    - This represents a test file that is responsible for testing `projectname/somefile.py`
 1. There are some other files: `__init__.py` and `context.py`
```

Comment:
GitHub won't let me select the first bullet on line 55 to suggest a change, but we should update it to acknowledge there's also a `venv` folder as well.

```suggestion
1. There are some other files such as `__init__.py`
```